### DISCUSSION TITLE
the HTML content of the first form is replaced with <wire:id="4v0vioSkkuZV8lzCQvrY">

### DISCUSSION THREAD

**Discussion by oussamaSikaddour** [livewire/livewire#7001] [1. Help]
i m encountering an issue where the HTML content of the first form is replaced with <wire:id="4v0vioSkkuZV8lzCQvrY"> during a form transition,from the frist form to the next , i m crate a two step register form , when a vlidate the first form , an transit to the next one , the code html change to <wire:id="4v0vioSkkuZV8lzCQvrY">, and mess my transtion

**Comment by joshhanley** [comment]
@oussamaSikaddour please share your code and also share some screenshots of the issue. Thanks!

**Reply by shokanshi** [reply]
I encounter similar behavior when I call the same livewire component recursively from within the child component itself.

Does Livewire v3 support recursive calling of the component from itself? This was working in v2.
